src/assets/close-icon.ts

```diff
@@ -0,0 +1,9 @@
+import { html } from 'lit-element';
```

Comment:
Yeah I was thinking about that but my concern is that I'm going to be importing an entire library of icons (and the file size that comes with it) for just this one close icon.

Reply:
I figured you'd pull the individual file directly, since you don't need a customElement for it. You should be able to install the package within this project, then import https://github.com/internetarchive/iaux/blob/master/packages/ia-icons/src/icons/close.js

Reply:
🤔 💭  can icons be a mono-repo with an option to import all and 1x1? is this our case for lerna? 😄 💭

Reply:
https://codesandbox.io/s/eager-sunset-v9cs7?file=/src/index.js

Reply:
But by adding `ia-icons` as an npm dependency, you're inherently bringing in the entire npm module with it, even though you're only using that one icon. When Pika packages the module, it creates a single file containing everything included in it, not just the imported files.

That's a lot of file-size overhead (that will continue to grow) for a single icon.

Reply:
The what is the solution? To not have an icons package? To have many identical copies of the same asset that, when design updates it, have to all be updated manually, leading to forgotten uses of the asset? This icon package will be included in the Petabox dependencies when ia-topnav switches to use it, and the plan was to also have the new BookReader chrome use it for the side menu icons. Since they're already in the codebase, there won't be any additional overhead when this is integrated.

Reply:
I like Isa's idea of a lerna monorepo of icons that can each be published individually and put the main `ia-icon` component in there that imports all of them. Then our components can pull them in piecemeal and we'll still have a central repo of all the icons.

Reply:
let's do it!

Reply:
So the solution is to have each SVG file be its own NPM package? That feels like overkill.

Reply:
I think they'd only be like 2 files each: `index.js` and `package.json`. We don't need tests or anything like that and since it'd be in lerna, we'd just have to make one good one then duplicate the directory and push the new icon.

Reply:
Now imported from iaux-icons